src/orders/groupOrder.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable import/no-cycle */
+import axios from "axios";
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/JoeMindo/USSDTest/pullRequest?prid=8695127)
```suggestion
import axios from 'axios';
```